parts/kubernetesparams.t

```diff
@@ -110,6 +110,90 @@
       },
       "type": "string"
     },
+    "kubernetesNodeStatusUpdateFrequency": {
+      {{PopulateClassicModeDefaultValue "kubernetesNodeStatusUpdateFrequency"}}
+      "metadata": {
+        "description": "Kubelet config for node status update frequency interval."
+      },
+      "type": "string"
+    },
+    "kubernetesCtrlMgrNodeMonitorGracePeriod": {
+      {{PopulateClassicModeDefaultValue "kubernetesCtrlMgrNodeMonitorGracePeriod"}}
+      "metadata": {
+        "description": "Kubernetes controller manager grace period for node status updates."
+      },
+      "type": "string"
+    },
+    "kubernetesCtrlMgrPodEvictionTimeout": {
+      {{PopulateClassicModeDefaultValue "kubernetesCtrlMgrPodEvictionTimeout"}}
+      "metadata": {
+        "description": "Kubernetes controller manager pod eviction timeout."
+      },
+      "type": "string"
+    },
+    "kubernetesCtrlMgrRouteReconciliationPeriod": {
+      {{PopulateClassicModeDefaultValue "kubernetesCtrlMgrRouteReconciliationPeriod"}}
+      "metadata": {
+        "description": "Kubernetes controller manager route reconciliation period."
+      },
+      "type": "string"
+    },
+    "cloudProviderBackoff": {
+      {{PopulateClassicModeDefaultValue "cloudProviderBackoff"}}
+      "metadata": {
+        "description": "Enable cloudprovider backoff?"
+      },
+      "type": "boolean"
```

Comment:
available types are 
string
secureString
int
bool
object 
secureObject
array,
So pick the apporiate one. for floats you might have to parse a string